src/crimson/osd/watch.h

```diff
@@ -53,4 +69,53 @@ class Watch : public std::enable_shared_from_this<Watch> {
 
 using WatchRef = Watch::Ref;
 
+class Notify {
+  std::set<WatchRef> watchers;
+
+  uint64_t get_id() { return 0; }
```

Comment:
could you add `const` specifier to this member function?

Reply:
TODO.